assets/grafana/celeborn-dashboard.json

```diff
@@ -4386,25 +4954,50 @@
                 "mode": "absolute",
                 "steps": [
                   {
-                    "color": "green"
+                    "color": "green",
+                    "value": null
                   },
                   {
                     "color": "red",
                     "value": 80
                   }
                 ]
-              },
-              "unit": "ms"
+              }
             },
-            "overrides": []
+            "overrides": [
+              {
+                "__systemRef": "hideSeriesFrom",
+                "matcher": {
+                  "id": "byNames",
+                  "options": {
+                    "mode": "exclude",
+                    "names": [
+                      "metrics_ActiveChunkStreamCount_Value {__name__=\"metrics_ActiveChunkStreamCount_Value\", exported_instance=\"10.230.216.22:9096\", instance=\"core-1-3:9096\", job=\"RSS\", role=\"worker\"}"
```

Comment:
could you revert the unnecessary changes?

Reply:
could you take a look at https://github.com/FMX/celeborn/pull/8?

Here is the commit that only add new metrics, https://github.com/FMX/celeborn/pull/8/commits/f666e59f35a4f76699d6c72ccdcb408e5364a642